src/api/data-migration/version-1-to-version-2/create-new-application-relationships/declaration-version-field.ts

```diff
@@ -18,7 +18,14 @@ const updateDeclarationVersionField = async (connection: Connection) => {
   console.info('✅ %s', loggingMessage);
 
   try {
-    const declarations = await getAllDeclarations(connection);
+    const declarations = await getAllDeclarationsNonSubmittedApplications(connection);
+
+    if (!declarations || !declarations.length) {
```

Comment:
```suggestion
    if (!declarations?.length) {
```

Reply:
This is not possible because `.length` cannot exist. `getAllDeclarationsNonSubmittedApplications` can return a boolean or an array.